core/src/test/java/discord4j/core/ExampleBot.java

```diff
@@ -186,8 +191,90 @@ public void testCommandBot() {
+                "🚡", "🚖", "🚘", "🚍", "🚔", "🚋", "🚝", "🚄", "🚅", "🚈", "🚞", "🚂", "🚆", "🛬", "🛫", "✈️", "🛩",
+                "🚁", "🚉", "🚊", "🚇", "⛵️", "🛥", "🚤", "⛴", "🚀", "🛳", "🛰", "💺", "🏁", "🚥", "🚦", "🚏", "⛽️",
+                "🚧", "⚓️", "🎡", "🎢", "🎠", "🏗", "🌁", "🗼", "🏭", "⛲️", "⛺️", "🏕", "🗾", "🌋", "🗻", "🏔", "⛰",
+                "🎑", "🏞", "🛣", "🛤", "🌅", "🌄", "🏜", "🏖", "🏝", "🎇", "🌠", "🌌", "🌉", "🌃", "🏙", "🌆", "🌇",
+                "🎆", "🌈", "🏘", "🏰", "🏯", "🏠", "🗽", "🏟", "🏡", "🏚", "🏢", "🏬", "🏣", "🏤", "🏥", "🏦", "🕌",
+                "🏛", "💒", "🏩", "🏫", "🏪", "🏨", "🕍", "🕋", "⛩", "🕹", "💽", "💾", "💿", "📼", "📷", "📹",
+                "🎥", "☎️", "⏱", "🎙", "📻", "📺", "📠", "📟", "⏲", "⏰", "🕰", "⏳", "📡", "🔋", "💴", "💵", "💸",
+                "🛢", "🔦", "💡", "💶", "💷", "💰", "💳", "💎", "🔨", "💣", "🔫", "🔪", "☠️", "🔮", "💈", "💊", "💉",
+                "🔖", "🚿", "🔑", "🛋", "🚪", "🛎", "🖼", "🎁", "🎀", "🎏", "🎈", "🛍", "⛱", "🗺", "🎊", "🎉", "🎎",
+                "🎐", "🎌", "🏮", "📮", "📫", "📯", "📊", "🗃", "📇", "📅", "📉", "📈", "📰", "📕", "📙", "📒", "✂️",
+                "🖇", "📖", "📚", "📌", "📍", "🚩", "❤️", "💔", "❣️", "💕", "💓", "💗", "💖", "💘", "💝", "💠", "🔔"};
+
+        @Override
+        public Mono<Void> onMessageCreate(MessageCreateEvent event) {
+            Message message = event.getMessage();
+            return message.getContent()
+                    .filter(content -> content.startsWith("!react"))
+                    .map(content -> {
+                        String rawCount = content.substring("!react".length());
+                        int count = 1;
+                        try {
+                            count = Math.max(1, Math.min(10, Integer.parseInt(rawCount.trim())));
+                        } catch (NumberFormatException e) {
+                            log.info("Could not parse {} into a number", e);
+                        }
+                        // Some emoji will fail with error 400, duplicates can happen, etc.
+                        return Flux.fromIterable(randomEmoji(count))
+                                .doOnNext(emoji -> log.info("Reacting with {}", emoji))
+                                .flatMap(emoji -> message.addReaction(ReactionEmoji.unicode(emoji)))
+                                .onErrorContinue(isStatusCode(400),
+                                        (t, o) -> log.info("Dropping value due to {}", t.toString()))
+                                .then();
+                    })
+                    .orElseGet(Mono::empty);
+        }
+
+        private List<String> randomEmoji(int count) {
+            List<String> emojiList = new ArrayList<>();
+            for (int i = 0; i < count; i++) {
+                emojiList.add(emoji[random.nextInt(emoji.length)]);
+            }
+            return emojiList;
+        }
+    }
+
     public static abstract class EventHandler {
 
         public abstract Mono<Void> onMessageCreate(MessageCreateEvent event);
     }
+
+    private static Predicate<Throwable> isStatusCode(int code) {
```

Comment:
Not yet, but we should discuss that separately, because there are also plans to provide a cross-cutting error strategy: https://github.com/Discord4J/Discord4J/compare/feature/on-status?expand=1